packages/app-desktop/gui/MainScreen/commands/showPrompt.ts

```diff
@@ -0,0 +1,28 @@
+import { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+
+export const declaration: CommandDeclaration = {
+	name: 'showPrompt',
+};
+
+export const runtime = (comp: any): CommandRuntime => {
+	return {
+		execute: async (_context: CommandContext, config: any) => {
```

Comment:
Would you mind defining an interface for the config? It's not too hard to do and it would make the command more usable. It would start like this:

```typescript
interface PromptConfig {
    label: string;
    description?: string;
    // etc.
}
```

If you search for `promptOptions:` in the codebase you'll find the possible properties.

Reply:
Sure, done.